docs/execution_profiles.rst

```diff
@@ -9,6 +9,10 @@ The legacy configuration remains intact, but legacy and Execution Profile APIs
 cannot be used simultaneously on the same client ``Cluster``. Legacy configuration
 will be removed in the next major release (4.0).
 
+Instances of execution profiles and the settings used by them should be unique across ``Cluster`` instances.
+For example, an execution profile should not be applied to more than one ``Cluster`` instance,
+and a ``LoadBalancingPolicy`` should not belong to more than one execution profile.
+
```

Comment:
suggestion:

`An execution profile and its parameters cannot be shared across multiple ``Cluster`` instances`. For example, an execution profile, its ``LoadBalancingPolicy`` and its ``RetryPolicy`` should not be applied to more than one ``Cluster`` instance.